### DISCUSSION TITLE
A few techniacl questios

### DISCUSSION THREAD

**Discussion by MODUNICO** [btcpayserver/btcpayserver#6145] [Technical Support] [answered]
1. How can I change the information on the Checkout (Point of Sale) payment? How to add another field to fill in? look the attachement, I have only 3 options to choose.
![image](https://github.com/user-attachments/assets/2821ed5a-7db8-4660-9ba3-52b1bce34afb)

2. I want to receive the donations in ETH or USDT? Explain me how to do it step by step please.
3. How to connect my Metamask / Trust wallet to my account? please explain me step bys step.

**Comment by Kukks** [comment, marked as answer]
Please take a moment to read our in-depth documentation next time. 

Forms: https://docs.btcpayserver.org/Forms/#setting-up-your-stores-first-custom-form
Altcoins: Install FixedFloat plugin
Meta/Trust: they are not bitcoin wallets and therefore not compatible.
